packages/strapi-admin/admin/src/components/Notifications/NotificationsContainer/Wrapper.js

```diff
@@ -11,6 +11,7 @@ const Wrapper = styled(TransitionGroup)`
   width: 300px;
   margin: 0 auto;
   overflow-y: hidden;
+  pointer-events: none;
```

Comment:
Can you also add the property to the `strapi-admin/admin/src/containers/NewNotification/Wrapper.js` so the bug to fix the issue with the new API also?

Reply:
Hi! Just done the change.